modules/workflow-service-impl/src/main/java/org/opencastproject/workflow/impl/WorkflowServiceImpl.java

```diff
@@ -1375,6 +1374,18 @@ public void update(final WorkflowInstance workflowInstance) throws WorkflowExcep
         throw new WorkflowDatabaseException(e);
       }
 
+      // get metadata for index update
+      String dcXml = null;
```

Comment:
This code seems worth to be factored out as a utility method as used two times in this file and also in https://github.com/opencast/opencast/pull/487/files#diff-9593c210ac7711725ecfed6cfefc9039R297

Reply:
Makes sense → updated.